src/components/layout/BaseLayout.tsx

```diff
@@ -1,16 +1,20 @@
 import React, { FC, PropsWithChildren, ReactNode } from 'react';
 
-import { chakra, Container, Flex } from '@chakra-ui/react';
+import { chakra, Container } from '@chakra-ui/react';
 
 import { sizes } from 'src/themes/shared/sizes';
+import { breakpoints } from 'src/themes';
+import { useMediaQuery } from 'src/hooks';
 
+import Flex from '../flex';
 import Divider from '../divider';
 
 interface Props {
   header?: ReactNode;
   footer?: ReactNode;
   skyScraperAd?: ReactNode;
   maxContentWidth: keyof typeof sizes.container;
+  isSrpAds?: boolean;
```

Comment:
this is very domain specific and should not be in the components library. IMO it is safe to just set the width to 500px for the xl breakpoint for all layouts. I wouldn't make that configurable at all.